cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -32,6 +32,7 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 	flags.String("memory-limit", "", "memory limit (e.g. 512m)")
 	flags.String("cpu-reservation", "", "number of CPU cores reserved (e.g. 0.5)")
 	flags.String("cpu-limit", "", "CPU cores limit (e.g. 0.5)")
+	flags.String(ThirdPartyResourceFlag, "", "user defined resources request (e.g. gpu=3;fpga=1)")
```

Comment:
It would make a lot of sense, thanks for pointing that out ! I was annoyed by the fact that there was a variable where all the others definitions used strings :)

Though we would have to agree on the name before changing that :)
- --generic-node-resource
- --generic-service-resource
could be the way to go if we follow the `GenericeResource` naming convention.

Do you have other naming suggestions ?